File: src/main/java/guess/GameFor.java
package guess;

import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        for (int i=1; i<=3; i++) {
            System.out.println(i);
        }

        for (int i=1; i<=5; i++) {
            Scanner scanner = new Scanner(System.in);
            int number = scanner.nextInt();
            System.out.println("第" + i + "次：" + number);
        }
    }
}

File: src/main/kotlin/com/courtney/GameFor.kt
package com.courtney

import java.lang.System.`in`
import java.util.*

fun main() {
    val scanner = Scanner(`in`)
    for (i in 1..5) {
        print("Please enter the number:")
        var number = scanner.nextInt()
        println("第${i}次：$number")
    }
}